skimage/draw/draw.py

```diff
@@ -190,11 +190,11 @@ def disk(center, radius, *, shape=None):
     radius : double
         Radius of disk.
     shape : tuple, optional
-        Image shape which is used to determine the maximum extent of output
-        pixel coordinates. This is useful for disks that exceed the image
-        size. If None, the full extent of the disk is used.  Must be at least
-        length 2. Only the first two values are used to determine the extent of
-        the input image.
+        Image shape as a tuple of size 2, used to determine the maximum
+        extent of output pixel coordinates. This is useful for disks that
+        exceed the image size. If None, the full extent of the disk is used.
+        The  shape might result in negative coordinates and wraparound
+        behaviour.
```

Comment:
I think that the confusing part of the wording was the `full extend of the disk is used` and the clarifying sentence makes sense but only 'cos I've seen where those negative indices are coming from.  I would keep this (or anything similar) but I would add a similar example to #5038 to compare `shape=None` to a tuple that matches shape `shape=(shape[0], shape[1])`

Reply:
Let me know what you think in the example introduced by 3076a61.

Reply:
Let me know what you think of the example introduced by 3076a61.